python/pyspark/mllib/common.py

```diff
@@ -113,16 +117,16 @@ def _java2py(sc, r, encoding="bytes"):
     return r
 
 
-def callJavaFunc(sc, func, *args):
+def callJavaFunc(sc: pyspark.context.SparkContext, func: Callable, *args: Any) -> JavaObjectType:
```

Comment:
If you want to keep the alias (https://github.com/apache/spark/pull/34671/files#r754559725, maybe with more descriptive name)  we could have t used here

`func: Callable[..., JavaObjectType]`?

Reply:
Not sure of a better alternative to `JavaObjectType` (which I agree is not a great name).

Maybe `JavaSerializedType`?

Reply:
Never mind, just saw your other comment.